Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.11.0-alpine3.7
+
+RUN apk update && \
+  apk upgrade --update-cache --available
+RUN apk add git make curl perl bash build-base zlib-dev ucl-dev
```

Comment:
Why we need perl? Did you just copy that from somewhere?

Reply:
Didn't notice perl. Will delete.

Reply:
Ok, but why we need every other dependency?

Reply:
Removed all unnecessary packages.